Overview.bs

```diff
@@ -342,23 +343,133 @@ response_type can be one of the following values:
 New Font Request {#rebase-request}
 ----------------------------------
 
+A new font request is sent by the client to get data for a font that it has not previously requested.
+In response the server will give the client a subset of the requested font that covers the codepoints
+listed in the request. A new font request is made via HTTP and may use either the GET or POST method.
+
+TODO(garretrieger): mention how the url identifies the specific font being requested.
+
+### POST ### {#rebase-request-post}
+
+If sent as a POST request the post body will be a single
+<a href="#PatchRequest"><code>PatchRequest</code></a> object encoded via CBOR. All fields of
+<a href="#PatchRequest"><code>PatchRequest</code></a> should be left unset except for:
+
+*  <a href="#PatchRequest"><code>PatchRequest.codepoints_needed</code></a>:<br/>
+    This field must be populated with the <a href="#CompressedSet">set</a> of
+    unicode codepoints which the client requires data for.
+   
+*  <a href="#PatchRequest"><code>PatchRequest.patch_format</code></a>:<br/>
+    This field should be populated with the set of <a href="#patch-formats">patch formats</a> that this
+    client is capable of decoding. At least one format must be provided.
+
+### GET ### {#rebase-request-get}
+
+If sent as a GET request the client will include a single query parameter:
+
+*  <code>request</code>:<br/>
+    The value is a single <a href="#PatchRequest"><code>PatchRequest</code></a> object encoded via CBOR
+    and then base64url encoding [[rfc4648]]. The <a href="#PatchRequest"><code>PatchRequest</code></a>
+    object uses the same fields as specified for a <a href="#rebase-request-post">POST</a> request.
+
 ### Standard Response ### {#rebase-standard-response}
 
+The server will response to a new font request with a <a href="#rebase-response">New Font Response</a>.
+
 ### Errors ### {#rebase-errors}
 
+The following errors may occur as a result of a new font request:
+
+*  [[#error-font-not-found]]
+*  [[#error-bad-request]]
+*  [[#error-internal-error]]
+
 Patch Font Request {#patch-request}
 -----------------------------------
 
+A patch font request is sent by a client to add data for additional codepoints to its font. Patch font
+requests are sent via HTTP and can only use the POST method. The request body is a single
+<a href="#PatchRequest"><code>PatchRequest</code></a> object encoded via CBOR. The fields of
+should be set as follows:
```

Comment:
I suspect the reference to an object (PatchRequest?) are missing in this sentence.